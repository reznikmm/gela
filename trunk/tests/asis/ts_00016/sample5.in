Associate -I../../../source/asis/spec -Isample sample/procedure_body-procedure_body_stub.adb
Action
Process_Context Compilation_Unit_Body    procedure_body.procedure_body_stub
