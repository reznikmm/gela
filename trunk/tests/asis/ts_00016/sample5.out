Declaration_Kind:A_PROCEDURE_BODY_DECLARATION Procedure_Body_Stub at procedure_body-procedure_body_stub.adb [2:1..5:4]
  Defining_Name_Kind:A_DEFINING_IDENTIFIER Procedure_Body_Stub at procedure_body-procedure_body_stub.adb [2:11..2:29]
  Declaration_Kind:A_PARAMETER_SPECIFICATION X at procedure_body-procedure_body_stub.adb [2:32..2:49]
    Defining_Name_Kind:A_DEFINING_IDENTIFIER X at procedure_body-procedure_body_stub.adb [2:32..2:32]
    Access_Definition_Kind:AN_ANONYMOUS_ACCESS_TO_VARIABLE  at procedure_body-procedure_body_stub.adb [2:36..2:49]
      Expression_Kind:AN_IDENTIFIER Integer at procedure_body-procedure_body_stub.adb [2:43..2:49]
  Statement_Kind:A_NULL_STATEMENT  at procedure_body-procedure_body_stub.adb [4:4..4:8]
